src/components/hmi_message_handler/CMakeLists.txt

```diff
@@ -27,65 +28,55 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
+include(${CMAKE_SOURCE_DIR}/tools/cmake/helpers/platform.cmake)
+include(${CMAKE_SOURCE_DIR}/tools/cmake/helpers/sources.cmake)
 
-include_directories (
-    ${COMPONENTS_DIR}/hmi_message_handler/include/
-    ${COMPONENTS_DIR}/application_manager/include/
-    ${COMPONENTS_DIR}/utils/include/
-    ${COMPONENTS_DIR}/smart_objects/include/
-    ${COMPONENTS_DIR}/formatters/include/
-    ${COMPONENTS_DIR}/config_profile/include/
-    ${COMPONENTS_DIR}/media_manager/include/
-    ${COMPONENTS_DIR}/protocol_handler/include
-    ${JSONCPP_INCLUDE_DIRECTORY}
-    ${MESSAGE_BROKER_INCLUDE_DIRECTORY}
-    ${CMAKE_BINARY_DIR}/src/components/
-    ${COMPONENTS_DIR}/dbus/include/
-    ${LOG4CXX_INCLUDE_DIRECTORY}
+include_directories(
+  ${COMPONENTS_DIR}/hmi_message_handler/include/
+  ${COMPONENTS_DIR}/application_manager/include/
+  ${COMPONENTS_DIR}/utils/include/
+  ${COMPONENTS_DIR}/smart_objects/include/
+  ${COMPONENTS_DIR}/formatters/include/
+  ${COMPONENTS_DIR}/config_profile/include/
+  ${COMPONENTS_DIR}/media_manager/include/
+  ${COMPONENTS_DIR}/protocol_handler/include
+  ${CMAKE_BINARY_DIR}/src/components/
+  ${COMPONENTS_DIR}/dbus/include/
+  ${JSONCPP_INCLUDE_DIRECTORY}
+  ${MESSAGE_BROKER_INCLUDE_DIRECTORY}
 )
 
-IF (HMIADAPTER STREQUAL "dbus")
-    set (DBUS_SOURCE ${COMPONENTS_DIR}/hmi_message_handler/src/dbus_message_adapter.cc)
-    set (DBUS_ADAPTER dbus)
-ENDIF ()
-
-set (SOURCES
-    ${COMPONENTS_DIR}/hmi_message_handler/src/hmi_message_handler_impl.cc
-    ${COMPONENTS_DIR}/hmi_message_handler/src/messagebroker_adapter.cc
-    ${COMPONENTS_DIR}/hmi_message_handler/src/hmi_message_adapter_impl.cc
-    ${DBUS_SOURCE}
+set(PATHS
+  ${CMAKE_CURRENT_SOURCE_DIR}/include
+  ${CMAKE_CURRENT_SOURCE_DIR}/src
 )
-if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
-  set(LIBRARIES
-    Utils
-    ${DBUS_ADAPTER}
-    ${RTLIB}
+
+set(EXCLUDE_PATHS)
+
+set(LIBRARIES
+  Utils
```

Comment:
@anosach-luxoft Why utils is still starts from upper-case if all libraries starts from lower?